src/promptflow-tools/tests/conftest.py

```diff
@@ -45,6 +45,24 @@ def serp_connection():
     return ConnectionManager().get("serp_connection")
 
 
+def verify_custom_connection(connection: CustomConnection) -> bool:
```

Comment:
How about renaming as `verify_oss_llm_custom_connection` to be more specific? An arbitrary custom connection may not have `endpoint_url`, then may go to line 63 `Task Succeeded unexpectedly.`